Source/WebCore/platform/win/ClipboardUtilitiesWin.cpp

```diff
@@ -167,7 +167,7 @@ HGLOBAL createGlobalData(const String& str)
     HGLOBAL vm = ::GlobalAlloc(GPTR, (str.length() + 1) * sizeof(UChar));
     if (!vm)
         return 0;
-    UChar* buffer = static_cast<UChar*>(GlobalLock(vm));
+    auto buffer = unsafeMakeSpan(static_cast<UChar*>(GlobalLock(vm)), str.length() + 1);
```

Comment:
I guess if we were serious about Windows we would make a GlobalAllocSpan data type. But it's probably not worth it for this patch.